packages/react-app/src/hooks/TokenList.js

```diff
@@ -41,7 +41,7 @@ const useTokenList = (tokenListUri, chainId) => {
       }
     };
     getTokenList();
-  }, [tokenListUri]);
+  }, [tokenListUri, _tokenListUri, chainId]);
```

Comment:
It seems this hook is not being used:

`views/Hints.jsx` uses the `useTokenList` hook but from `eth-hooks/dapps/dex`, not this one.

I tested it anyway, and it works ;)